rpcserver.go

```diff
@@ -2455,6 +2455,14 @@ func (r *rpcServer) BatchOpenChannel(ctx context.Context,
 		return nil, fmt.Errorf("specify at least one channel")
 	}
 
+	_, err := lnrpc.UnmarshallCoinSelectionStrategy(
+		in.CoinSelectionStrategy,
+		r.cfg.CoinSelectionStrategy,
+	)
+	if err != nil {
+		return nil, err
```

Comment:
unused?

Reply:
Yes forgot to remove it when used coin selection strategy as string in the beginning 👍